jovian/__init__.py

```diff
@@ -207,5 +207,20 @@ def log_metrics(data, verbose=True):
     global _data_blocks
     res = post_block(data, 'metrics')
     _data_blocks.append(res['tracking']['trackingSlug'])
-    if(verbose):
+    if verbose:
         log('Metrics logged.')
+
+
+def notify(data, verbose=True):
+    """Sends the data to Slack connected to Jovian account
+
+    Arguments:
+        :param data: A dict or string of dicts to be pushed to Slack
```

Comment:
we don't use this format for specifying arguments. Check the other methods above.